xrootd.spec

```diff
@@ -1,17 +1,18 @@
-### RPM external xrootd 5.6.2
+### RPM external xrootd 5.7.2
 ## INITENV +PATH LD_LIBRARY_PATH %i/lib64
 ## INITENV +PATH PYTHON3PATH %{i}/${PYTHON3_LIB_SITE_PACKAGES}
 
 %define strip_files %i/lib
 %define tag %{realversion}
 %define branch master
 %define github_user xrootd
-Source: https://xrootd.slac.stanford.edu/download/v%{realversion}/%{n}-%{realversion}.tar.gz
+Source: https://github.com/xrootd/xrootd/releases/download/v%{realversion}/%{n}-%{realversion}.tar.gz
 
 BuildRequires: cmake gmake autotools py3-pip
 Requires: zlib libuuid curl davix
 Requires: python3 py3-setuptools
 Requires: libxml2
+Requires: isal
```

Comment:
@iarspider , do we need to add `${ISAL_ROOT}` in to CMAKE_PREFIX_PATH ?

Reply:
Yes. I am running a local test now.